handlers/validate/steplib/version.go

```diff
@@ -0,0 +1,22 @@
+package steplib
+
+import (
+	stepmanModels "github.com/bitrise-io/stepman/models"
+)
+
+type Version struct {
+	ID          string
+	Previous    *Version
```

Comment:
unused property

Reply:
It is a preparation for the next validator task(diff the whole code change(not only the step.yml) and find important changes)

Reply:
could you please link where do you use this property?

Reply:
I started to develop this feature 2 weeks ago here: https://github.com/bitrise-io/bitrise-steplib/tree/diff-tool and it is not yet updated to use the new StepLib model. (Which is in this PR)